ndm-operator.yaml

```diff
@@ -180,6 +178,9 @@ spec:
         hostPath:
           path: /proc
           type: Directory
+      - name: basepath
+        hostPath:
+          path: /var/openebs/ndm
       - name: sparsepath
```

Comment:
we need to change helm also, right? We should remove sparsepath and add basepath in the helm?

Reply:
The changes to helm will be done in [this](https://github.com/openebs/openebs/pull/2880) PR in Openebs